static/app/components/themeAndStyleProvider.tsx

```diff
@@ -0,0 +1,39 @@
+import {useEffect} from 'react';
+import ReactDOM from 'react-dom';
+import {cache} from '@emotion/css'; // eslint-disable-line emotion/no-vanilla
+import {CacheProvider, ThemeProvider} from '@emotion/react'; // This is needed to set "speedy" = false (for percy)
```

Comment:
Is this comment still up to date now that we don't use percy?

Reply:
good question cc @billyvg

Reply:
Hmm, I don't remember -- it might be needed for jest snapshots.